### ISSUE TITLE
RuntimeException crash

### ISSUE THREAD

**Issue by YukiRainee** [TeamPneumatic/pnc-repressurized#1251]
### Describe the bug

Attempting to launch pneumaticcraft with neoforge 
47.1.79 crashes the game on startup with an error showing 'java.lang.RuntimeException: Mod fluids must override getFluidType' as shown in the screenshot below.

### How to reproduce the bug

Install pneumaticcraft to a profile with neoforge 
47.1.79 minecraft version 1.20.1 installed and attempt to launch
it should crash with an error somewhat similar in the pictures linked below describing that 'mod fluids must override getFluidType'

### Expected behavior

Start up to minecraft main menu, and able to create world

### Additional details

https://i.imgur.com/jHsQrKX.png
https://mclo.gs/8lW5OWE

I'm not too sure if NeoForge is currently supported but I wanted to make this report just in case thanks for doing everything that you do!

### Which Minecraft version are you using?

1.20

### Which version of PneumaticCraft: Repressurized are you using?

6.0.10+mc1.20.1

### Crash log

https://mclo.gs/zDlGZ3w

**Comment by desht** [comment]
This isn't a bug in PneumaticCraft, but in some other mod which has forgotten to register a fluid type for their modded fluid. It's just PNC which first tries to get the fluid type for the fluid, and cause Forge to throw an exception.

This is tricky to track down, but I can add some code to PNC to catch that exception and report the fluid in question. If I do that, can you try out a dev build of PNC, and let me know what gets reported in the log file?

**Comment by YukiRainee** [comment]
Yeah sure I'm willing to do that I am also in the discord if it needs to be talked about more just need to know where to download the mod file

**Comment by desht** [comment]
OK, the dev build is https://modmaven.dev/me/desht/pneumaticcraft/pneumaticcraft-repressurized/6.0.11+mc1.20.1-SNAPSHOT/pneumaticcraft-repressurized-6.0.11+mc1.20.1-20231109.164823-2.jar

and yeah feel free to discuss in Discord if you need :)  mainly just interested to see which fluid appears in the error log.  Look for a message like `Caught exception while checking the fluid type of`
